<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>392</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>色域检测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/icon.ico</normaloff>:/img/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton_CheckDevice">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>345</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>检测设备</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>型        号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_EdidVersion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>EDID版本:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Gamut">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>生产日期:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>色        域:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ProductModel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_ProductDate">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>设        备:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Device">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>13</y>
       <width>120</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_GreenY">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>29</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>53</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue    Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_WhiteY">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>78</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_BlueX">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>53</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>78</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>White Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>53</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue   X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GreenX">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>29</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_WhiteX">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>78</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_BlueY">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>53</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_RedX">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>5</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>29</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Green Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_RedY">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>5</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>29</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Green X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>5</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Red     Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Red     X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>78</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>White X:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_AboutMe">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>345</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
